Rakefile

```diff
@@ -145,4 +145,56 @@ namespace :site do
       abort "You seem to have misplaced your History.markdown file. I can haz?"
     end
   end
+
+  desc "generate importer-dependencies data file"
+  task :generate_dependency_data do
```

Comment:
Should we add this task as a dependency to the site build/generate task if there is one?